e2e/systemTest.spec.js

```diff
@@ -88,6 +91,7 @@ describe("Main screen", () => {
     await element(by.id("search")).tap();
     await element(by.id("search")).typeText("hall building");
     await element(by.id("search")).tapAtPoint({ x: 200, y: 75 });
```

Comment:
We will need a real fix instead of the workaround (clicking on x and y coordinates). We will need to find to way to fetch and get the button component directly inside the view as opposed as getting the view and clicking at a precise point. This workaround is fine for now cause it tests the functionality of our app. However, it isn't a good solution for extension and modification/maintenance properties of the code.